README.md

```diff
@@ -18,6 +18,10 @@ a text editor, eventually...
 I used to use the text editor `nano`, frequently I typo this.  on a qwerty
 keyboard, when the right hand is shifted left by one, `nano` becomes `babi`.
 
+### babi vs nano
+
+[![youtube-video-about-babi](https://img.youtube.com/vi/WyR1hAGmR3g/0.jpg)](https://www.youtube.com/watch?v=WyR1hAGmR3g&feature=youtu.be&t=193)
```

Comment:
hmmm the thumbnail makes it kinda difficult to see that this is a youtube video, let's maybe do the following things:

1. adjust the title of this section to "babi vs. nano" (add a dot)
2. add this copy: "here is a youtube video where I discuss the motivation for creating and using babi instead of nano:"
3. use the youtube short link `https://youtu.be/WyR1hAGmR3g`
4. this thumbnail is ~slightly better: `https://img.youtube.com/vi/WyR1hAGmR3g/mqdefault.jpg`

a preview:

___

### babi vs. nano

here is a youtube video where I discuss the motivation for creating and using babi instead of nano:

[![youtube video about babi](https://img.youtube.com/vi/WyR1hAGmR3g/mqdefault.jpg)](https://youtu.be/WyR1hAGmR3g)